boxoffice/views/admin_orders.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+from flask import jsonify, g, url_for
+from decimal import Decimal
+from .. import app, lastuser
+from sqlalchemy import func
+from coaster.views import load_models, render_with
+from boxoffice.models import db, ItemCollection, Order, LineItem, LINE_ITEM_STATUS, CURRENCY_SYMBOL, ORDER_STATUS
+from boxoffice.views.order import jsonify_assignee
+from utils import invoice_date_filter
+
+
+def tickets_assignment_complete(order):
```

Comment:
As per #75, everything should be a ticket henceforth. We should deprecate use of the term "Item".